### ISSUE TITLE
Create More Tests with Complex Data

### ISSUE THREAD

**Issue by ozdemirburak** [ozdemirburak/json-csv#3]
Probably this package has some bugs that can be identified with new tests. With taking inspiration from the current tests located in [CsvTest.php](https://github.com/ozdemirburak/json-csv/blob/master/tests/CsvTest.php) and [JsonTest.php](https://github.com/ozdemirburak/json-csv/blob/master/tests/JsonTest.php) files, you may create or modify some tests with more complex data.

What I mean by complex data is nested data like the below one.

```json
{
  "a": {
    "b": {
      "c": 1,
      "d": {
        "e": [
          {
            "f": 2,
            "g": 3
          }
        ]
      }
    }
  }
}
```

**TLDR:** Create new tests to make package fail.
